src/index.css

```diff
@@ -15,6 +15,10 @@
     max-width: 570px;
   }
 
+  .content-min-height {

```

Comment:
Isn't this is same as css in pageSkeleton

Reply:
Move from css scoped of pageSkeleton to global krub

Reply:
So that i can use it in other skeleton if needed (for future org skeleton)